src/main/java/com/agilethought/atsceapi/domain/FixedExpenses.java

```diff
@@ -0,0 +1,15 @@
+package com.agilethought.atsceapi.domain;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class FixedExpenses {
+    private double rent;
+    private double transport;
+    private double internet;
+    private double feed;
+    private double others;
+    private double total;
+}
```

Comment:
Shouldn't this be a model and not a domain?

Reply:
I have built this class in the domain package because it has internal usage (in the backend) for the most part. I mean, it will always belong to the History class and will not be used as an independent collection in the DB, so I considered more appropriate to define it as a POJO inside the domain package. Anyway, if it needs to be a model, I can do that change ASAP.

Reply:
Good points, yeah. What do you think @ivanflores15 and @EmiliaGuillen-dev? Can this stay as is or should it be changed?

Reply:
It's a good reason, I'm ok to left it into domain package